lib/grinder_sdk.dart

```diff
@@ -33,6 +33,51 @@ Directory getSdkDir([List<String> cliArgs]) => cli_util.getSdkDir(cliArgs);
 
 File get dartVM => joinFile(sdkDir, ['bin', _sdkBin('dart')]);
 
+/// Utility tasks for for getting information about the Dart SDK and invoking
+/// dart applications.
+class Dart {
```

Comment:
Hmm, it might be interesting to namespace all the sdk stuff, so:

```dart
class Sdk {

  /// Defaults the path to the current SDK.
  Sdk([String path]);
  
  static Directory get dir;

  final Vm vm;
  final Compiler compiler;
  final Analyzer analyzer;
  final Formatter formatter;
}

final Sdk sdk = new Sdk();
```

```dart
print('Sdk dir: ${sdk.dir}');
sdk.vm.run(...);
sdk.compiler.compile(...);
sdk.analyzer.analyze(...);
sdk.formatter.format(...);
```